test/secretbuffer.jl

```diff
@@ -80,4 +80,15 @@ using Test
         @test read(sb, String) == "\xff\xff\xff"
         shred!(sb)
     end
+    @testset "testing the skip function" begin
+    sb = SecretBuffer("computer")
+    skip(sb, 2)
+    a = position(sb)
+    seek(sb,0)
```

Comment:
Spacing. I would suggest spaces after commas in function calls here.

Reply:
Done! Changed it in the new commit!